lib/generators/rolemodel/saas/devise/templates/app/views/devise/sessions/new.html.slim

```diff
@@ -8,8 +8,8 @@
         .form__group
           = f.label :remember_me
             = f.check_box :remember_me
-            span.margin-left-xs Remember me
+            span.margin-left-xs= t("activerecord.attributes.user.remember_me", default: "Remember me")
```

Comment:
This could probably be replace with a `f.input` since we are using SimpleForm right?

Reply:
Sure, we can change that. Could you approve the simple_form PR (https://github.com/RoleModel/rolemodel_rails/pull/39)? 😉

Reply:
fixed